doc/siterc.tex

```diff
@@ -853,3 +853,70 @@ \subsection{[authentication]}
     \end{myitemize}
 \item {\em default:} state-totals
 \end{myitemize}
+
+\subsubsection{[authentication] $\rightarrow$ hashes}
+
+This sets the hash algorithms used for Pyro HMAC. SHA-256 and SHA-512 are
+the most secure. The first item is used as the default hash. Subsequent items
+(if any) are used as fallback hashes (e.g. for backwards compatibility should
```

Comment:
Something like this might be a bit clearer? - "for backwards compatibility *with older suite daemons that are still running*"